20_classes_lab/01.py

```diff
@@ -1,6 +1,30 @@
-"""
-The following code assumes a class
-named Summer exists. 
-Complete the class so correct result is printed
-"""
-
+"""
+Elad Agam - Classes 
+01 - Build Class summer
+"""
+
+class Summer(object):
+
+    def __init__(self):
+        self.sum = 0
+    
+    def add(self, number1,number2=0):
+        self.sum = self.sum + number1 + number2
+
+    def total(self):
+        return str(self.sum)
+
+
+
+s = Summer()
+t = Summer()
+
+s.add(10, 20)
+t.add(50)
+s.add(30)
+
+# should print 60
+print s.total()
+
+# should print 50
+print t.total()
```

Comment:
Done!

On 26 July 2016 at 09:17, Gabor Szabo <notifications@github.com> wrote:

> In 20_classes_lab/01.py
> <https://github.com/ynonp/python-examples-verint-2016-07/pull/209#discussion_r72192966>
> :
>
> > +        return str(self.sum)
> > +
> > +
> > +
> > +s = Summer()
> > +t = Summer()
> > +
> > +s.add(10, 20)
> > +t.add(50)
> > +s.add(30)
> > +
> > +# should print 60
> > +print s.total()
> > +
> > +# should print 50
> > +print t.total()
>
> Could you allow for any number of parameters in the 'add' method so I can
> call s.add(2, 5,7, 23, 13) as well?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ynonp/python-examples-verint-2016-07/pull/209/files/fa31a01d2d72fe87aba9177d9d3afd99681e13de#r72192966>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARMSUN_IBJWLNDGiu-gs4I4w5NEygKkJks5qZaZqgaJpZM4JUO0H>
> .
>